example.json

```
@@ -41,6 +41,10 @@
                     "alt": "",
                     "attribution": "Copyright © 2019",
                     "_classes": ""
+                },
+                "_tooltip": {
+                    "_isEnabled": false,
+                    "text": ""
```

Please add this to all example items:
```suggestion
                    "text": "{{title}}"
```